cli/cli.py

```diff
@@ -14,14 +14,13 @@
 #
 import click
 
-from cli.create_legacy_catalog import create_legacy_catalog_cli
-from cli.function_to_item import function_to_item_cli
-from cli.item_to_function import item_to_function_cli
+from cli.functions.function_to_item import function_to_item_cli
+from cli.functions.item_to_function import item_to_function_cli
 from cli.marketplace.build import build_marketplace_cli
-from cli.new_item import new_item
-from cli.test_suite import test_suite
-from cli.item_yaml import update_functions_yaml
-
+from cli.functions.new_function_item import new_item
```

Comment:
maybe consider to rename `new_item` method to `new_function_item` just to keep consistency when we add more items